crates/exex/Cargo.toml

```diff
@@ -12,10 +12,17 @@ description = "Execution extensions for Reth"
 workspace = true
 
 [dependencies]
+eyre.workspace = true
+futures.workspace = true
+metrics.workspace = true
 reth-config.workspace = true
+reth-metrics.workspace = true
 reth-node-api.workspace = true
 reth-node-core.workspace = true
 reth-primitives.workspace = true
 reth-provider.workspace = true
 reth-tasks.workspace = true
+reth-tracing.workspace = true
 tokio.workspace = true
+tokio-stream.workspace = true
+tokio-util.workspace = true
```

Comment:
nit: can we sort and annotate with comments deps here same as in other crates? reth first, then others